gsk4/Gir.toml

```diff
@@ -142,6 +197,16 @@ status = "generate"
     name = "free_to_args"
     ignore = true # C convenience functions
 
+[[object]]
+name = "Gsk.ShadowNode"
+status = "generate"
+    [[object.function]]
+    name = "new"
+    manual = true # can't work due to &[&CopyType] issue
```

Comment:
Wasn't this solved?

Reply:
Yes, but in a different merge request, whatever lands first on Gir's side